composer.json

```diff
@@ -11,5 +11,31 @@
 		"sirbrillig/phpcs-variable-analysis": "2.8.3",
 		"sirbrillig/phpcs-no-get-current-user": "1.0.1"
 	},
-	"prefer-stable": true
+	"prefer-stable": true,
+	"archive": {
+		"exclude": [
+			"/*",
```

Comment:
I switch this around to match our previous configuration. It also feels a tiny bit safer in some ways and more dangerous in others (especially in the asset directory).